cockatrice/src/shortcutssettings.h

```diff
@@ -473,24 +476,34 @@ class ShortcutsSettings : public QObject
         {"Player/aViewSideboard", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Sideboard"),
                                               parseSequenceString("Ctrl+F3"),
                                               ShortcutGroup::View)},
-        {"Player/aViewTopCards", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Top cards of library"),
+        {"Player/aViewTopCards", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Top Card(s) of Library"),
```

Comment:
The function is called "View top cards of library..." in the right click menu, we use (s) in cases where the s gets added depending on a variable.

Reply:
Yeah my B